crates/uv-cli/src/lib.rs

```diff
@@ -87,6 +87,7 @@ pub struct Cli {
 }
 
 #[derive(Parser, Debug, Clone)]
+#[command(mut_args(|arg| arg.help_heading("Global Options")))]
```

Comment:
Nit: I don't think we use title case

```suggestion
#[command(mut_args(|arg| arg.help_heading("Global options")))]
```